pkg/cli/env.go

```diff
@@ -188,10 +196,7 @@ func EnvSet(rack sdk.Interface, c *stdcli.Context) error {
 		c.Writer().Stdout = c.Writer().Stderr
 	}
 
-	env, err := helpers.AppEnvironment(rack, app(c))
-	if err != nil {
-		return err
-	}
+	env, err := loadEnvs(rack, c, c.Bool("replace"))
```

Comment:
This is no longer checking the error return value.

I think you don't need the extra function here, maybe something like:

```golang
env := structs.Environment{}
var err error

if !c.Bool("replace") {
  env, err = helpers.AppEnvironment(rack, app(c))
  if err != nil {
    return err
  }
}